examples/psychophysics.py

```diff
@@ -102,6 +102,6 @@
     plt.plot(X_true[:, 0], X_true[:, 1], 'or', label='SOE', ms=10)
     text_args = dict(fontsize=9, horizontalalignment='center', verticalalignment='center')
     for i in range(n_objects):
-        plt.text(*X_est[i] , str(i), **text_args)
-        plt.text(*X_true[i] , str(i), **text_args)
+        plt.text(X_est[i, 0], X_est[i, 1], str(i), **text_args)
```

Comment:
Thanks! However, the example would still be broken for n_dim==1. Could be fixed by changing line 96
to `X_est.shape[1] == 2`